### ISSUE TITLE
Improve documentation

### ISSUE THREAD

**Issue by bastula** [dicompyler/dicompyler-core#37]
Documentation with more examples would be great.

Ideally docstrings need to be completed for the `dvhcalc` module as well.

**Comment by alexandrehuat** [comment]
Hi, there's an issue in https://dicompyler-core.readthedocs.io/en/latest/installation.html
It's written `pip install dicompylercore` instead of `dicompyler-core`.

**Comment by bastula** [comment]
Thanks for that catch. I have just fixed it on master in commit f5f9fa770d0573373510135863fa2933919cdba7.

**Comment by peterfortuin** [comment]
An example how to create a DVH object from a dose en volume arrays would be nice. I have trouble with it.

**Comment by bastula** [comment]
@peterfortuin You're correct, there is no explicit documentation in the published docs or in the Jupyter notebook to show how to do that. However, you can follow the test suite and the following functions `test_raw_data_dvh` and `test_raw_data_dvh_max_bins` show how to accomplish this:

https://github.com/dicompyler/dicompyler-core/blob/64180836680a95554b36ed2b182eb9eb18ccdfb4/tests/test_dvh.py#L38

https://github.com/dicompyler/dicompyler-core/blob/64180836680a95554b36ed2b182eb9eb18ccdfb4/tests/test_dvh.py#L48

**Comment by bastula** [comment]
Improve docstrings (including function parameters and return values for) the `dvhcalc` module per: https://groups.google.com/g/dicompyler/c/Peil-ziqX_c/m/N76sg5kXAgAJ
